pkg/s3/s3_service.go

```diff
@@ -119,7 +119,11 @@ func (ps3 *S3Client) UploadFile(ctx context.Context, bucket string, filename str
 		Body:   file,
 	}
 
-	uploader := manager.NewUploader(ps3.Client)
+	// The upload manager breaks large data into parts and uploads the parts concurrent
+	// each part is 1mb
+	uploader := manager.NewUploader(ps3.Client, func(u *manager.Uploader) {
```

Comment:
should this be the default or have a separate function like UploadLargeFile() which does the chunking. Any cons with this approach